numpy/lib/tests/test_utils.py

```diff
@@ -51,6 +78,23 @@ def test_deprecate_fn():
     assert_('new_func3' in new_func3.__doc__)
 
 
+def test_deprecate_help_indentation():
+    assert_('    ' not in inspect.getdoc(old_func4))
+    assert_('    ' not in inspect.getdoc(old_func5))
+    assert_('    ' not in inspect.getdoc(old_func6))
+
+
+@dec.skipif(sys.flags.optimize == 2)
```

Comment:
What am I missing here - why does only this test need this?

Reply:
Optimization discards docstrings and presence of strings is checked in this case, `python3 -OO runtests.py -s lib.tests.test_utils` fails without the `skipif` (added a `reason` to the decorator).

Reply:
Thanks!

Reply:
Don't use `dec` methods, they are nose artifacts kept for downstream compatibility. Use `pytest.mark.skipif` instead. It will also require a `reason=` argument.